index.js

```diff
@@ -163,9 +185,9 @@ exports.verify = function(next, connection) {
     opendkim.query_info(this.cfg.general.query_info);
   }
 
-  opendkim.verify({id: txn.uuid});
+  opendkim.verify_sync({id: txn.uuid});
```

Comment:
I might still change this to an async call, but under the hood, there appears to be no blocking code.